### DISCUSSION TITLE
(solved) Oh snap! You got an error! Websu UI

### DISCUSSION THREAD

**Discussion by jarac794** [websu-io/websu#19] [General]
install the Websu UI but to analyze a website it throws me this error, any possible solution?

Oh snap! You got an error!
<!DOCTYPE html> <html lang="en"> <head> <meta charset="utf-8"> <title>Error</title> </head> <body> <pre>Cannot POST /reports</pre> </body> </html>

**Comment by samos123** [comment]
I am unable to reproduce this. Please post steps to reproduce. I suspect the issue is that you arent running the websu api service. You can try running with docker compose

**Comment by jarac794** [comment]
Thanks, I was able to solve it I was working on the interface but I didn't have the api configured
